src/Aspire.Dashboard/Components/Layout/MainLayout.razor

```diff
@@ -73,7 +73,7 @@
     <FluentDialogProvider/>
     <FluentTooltipProvider />
     @* Temporary work around for https://github.com/microsoft/fluentui-blazor/issues/2736 *@
-    <div style="position: fixed; bottom: 0;">
+    <div style="position: fixed; bottom: 0; z-index: 9900;">
```

Comment:
9900 is copied from FluentUI popup overlay.

Reply:
This comment would be good in the code itself for future reference.

Reply:
This should be temporary